sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClientBuilderExtensions.cs

```diff
@@ -4,15 +4,44 @@
 using System;
 using Azure;
 using Azure.AI.FormRecognizer;
+using Azure.AI.FormRecognizer.DocumentAnalysis;
 using Azure.Core.Extensions;
 
 namespace Microsoft.Extensions.Azure
 {
     /// <summary>
-    /// Extension methods to add <see cref="FormRecognizerClient"/> client to clients builder.
+    /// Extension methods to add <see cref="FormRecognizerClient"/> and <see cref="DocumentAnalysisClient"/> clients
+    /// to clients builder.
```

Comment:
Since Form Recognizer now have two main separate clients (the old `FormRecognizerClient` and the new `DocumentAnalysisClient`), we now have conflicting guidelines:
> DO provide a single *ClientBuilderExtensions class for every Azure SDK client library that contains client types. Name of the type should use the same prefix as the *ClientOptions class used across the library.

We now have two separate `ClientOptions` classes used across the library (`FormRecognizerClientOptions` and `DocumentAnalysisClientOptions`). We could have:
- Two separate extension files, `FormRecognizerClientBuilderExtensions` and `DocumentAnalysisClientBuilderExtensions`, each one for a different client.
- A single `FormRecognizerClientBuilderExtensions` file for both clients.

I opted for the second approach. Developers in our team are expecting a single file per package. I'm afraid one of them could end up being overlooked in the future and missing important changes.

Reply:
Agreed on the approach; I'd rather have the once class given that both sets of registrations serve the same purpose.